app/celery/scheduled_tasks.py

```diff
@@ -231,3 +236,45 @@ def check_for_services_with_high_failure_rates_or_sending_to_tv_numbers():
                 technical_ticket=True,
             )
             zendesk_client.send_ticket_to_zendesk(ticket)
+
+
+@notify_celery.task(
+    bind=True, max_retries=7, default_retry_delay=3600, name="process-delivery-receipts"
```

Comment:
Well, we won't know until we try.   :)

We have to retry in case we go down for any reason, don't want to lose receipts permanently.  But I don't think we have to retry that hard.  We will never get throttled on this call, because throttling only happens at 5x per second.

Reply:
I see, so this is in the case of a retry, not during the normal scheduled run every 8 minutes.  Thanks!